app/helpers/controllers/validateOrderItemFormBulk.test.js

```diff
@@ -169,10 +169,10 @@ describe('validateOrderItemFormBulk', () => {
       expect(errors).toEqual([priceLessThanMax]);
     });
 
-    it('should return an array of one validation error if price is greater than the list price', () => {
+    it('should return an array of one validation error if price is greater than the list price and no orignal price exists', () => {
       getSelectedPriceManifest.getSelectedPriceManifest.mockReturnValue(selectedPriceManifest);
       const data = {
-        price: '0.11',
+        price: '1.1',
```

Comment:
As above, I'm not sure this is correct. If we don't have a list price we can't perform the validation but that should never occur as the order form won't work without an available list price.